### DISCUSSION TITLE
Default theme improvement

### DISCUSSION THREAD

**Discussion by anytizer** [opencart/opencart#10938] [General]
The featured product list should be visually aligned to 3 lines for description, yielding a harmonized look/feel. User may modify their content definitely, but I am talking about improving the default theme which might remain active in production.

![image](https://user-images.githubusercontent.com/5563341/170884141-e84e9785-8989-44cf-b4a1-944dd72d5ab1.png)

**Comment by felipoantonoff** [comment]
Hi, in this case maybe a min-h CSS rule is enough to always stay in your good case for a height of 3 lines or less based on your example.

It might also be better to limit it to 2 lines in this description, limiting based on the amount of characters.

**Comment by anytizer** [comment]
![image](https://user-images.githubusercontent.com/5563341/171050998-602f00a1-f1a4-4156-94f4-2f97cd7e2459.png)
It appears to be responsive with different height in smaller screen.

Can someone please suggest a css fix?

**Reply by felipoantonoff** [reply]
It uses media queries, so for mobile as the screen resolution sets a different minimum height.

I recommend basing yourself on the most common cell phone resolutions or creating 2-3 rules if necessary.
